stable/jenkins/values.yaml

```diff
@@ -97,10 +97,14 @@ master:
   # Enable Kubernetes Liveness and Readiness Probes
   # ~ 2 minutes to allow Jenkins to restart when upgrading plugins. Set ReadinessTimeout to be shorter than LivenessTimeout.
   healthProbes: true
-  healthProbesLivenessTimeout: 90
-  healthProbesReadinessTimeout: 60
-  healthProbeReadinessPeriodSeconds: 10
-  healthProbeLivenessFailureThreshold: 12
+  healthProbesLivenessTimeout: 20
+  healthProbesReadinessTimeout: 20
+  healthProbeLivenessPeriodSeconds: 30
```

Comment:
I suggest to change `healthProbeLivenessPeriodSeconds` to 10 as we didn't have it before and that seems to be the kubernetes default:

`kubectl explain deployment.spec.template.spec.containers.readinessProbe.periodSeconds

>    periodSeconds        <integer>
>      How often (in seconds) to perform the probe. Default to 10 seconds. Minimum
>      value is 1.